static_analyzer/incremental_orchestrator.py

```
@@ -184,6 +198,78 @@ def _restore_cross_boundary_edges(
+            queries.append((Path(str(site["file"])), int(site["line"]) - 1, int(site["column"]) - 1))
+            lookup.append((edge, site))
+
+    confirmed: dict[tuple[str, str], list[dict[str, str | int]]] = {}
+    for start in range(0, len(queries), _DEFINITION_BATCH_SIZE):
+        batch = lookup[start : start + _DEFINITION_BATCH_SIZE]
+        try:
+            definition_results, _ = engine_client.send_definition_batch(queries[start : start + _DEFINITION_BATCH_SIZE])
+        except Exception:
+            logger.debug("Definition batch failed while restoring cached edges", exc_info=True)
+            continue
+        for (edge, site), definitions in zip(batch, definition_results):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)